memory error while installing php 8.0.19

I am getting this error:

```

Fatal error: Uncaught InvalidArgumentException: RegexIterator::__construct(): Allocation of JIT memory failed, PCRE JIT will be disabled. This is likely caused by security restrictions. Either grant PHP permission to allocate executable memory, or set pcre.jit=0 in /private/var/folders/rg/0z_ltnpx3fl9mtcqf9y6y5640000gp/T/php-src-php-8.0.19/ext/phar/phar.php:1133
Stack trace:
#0 /private/var/folders/rg/0z_ltnpx3fl9mtcqf9y6y5640000gp/T/php-src-php-8.0.19/ext/phar/phar.php(1133): RegexIterator->__construct(Object(RecursiveIteratorIterator), '/\\.svn/')
#1 /private/var/folders/rg/0z_ltnpx3fl9mtcqf9y6y5640000gp/T/php-src-php-8.0.19/ext/phar/phar.php(1077): PharCommand::phar_add(Object(Phar), 0, '/var/folders/rg...', NULL, '/\\.svn/', Object(SplFileInfo), NULL, true)
#2 /private/var/folders/rg/0z_ltnpx3fl9mtcqf9y6y5640000gp/T/php-src-php-8.0.19/ext/phar/phar.php(225): PharCommand->cli_cmd_run_pack(Array)
#3 /private/var/folders/rg/0z_ltnpx3fl9mtcqf9y6y5640000gp/T/php-src-php-8.0.19/ext/phar/phar.php(2089): CLICommand->__construct(19, Array)
#4 {main}
  thrown in /private/var/folders/rg/0z_ltnpx3fl9mtcqf9y6y5640000gp/T/php-src-php-8.0.19/ext/phar/phar.php on line 1133
make: *** [ext/phar/phar.phar] Error 255
make: *** Waiting for unfinished jobs....
```

everything i've found online indicates I need to alter a php.ini setting to fix this, but i'm not sure how to do that before php is installed. Also, could it be that I simply don't have enough memory on my computer? I am runnig an M1 with 16GB RAM.

--- a/bin/install
+++ b/bin/install
@@ -81,6 +81,12 @@ install_php() {
 
     # Target is OS-specific
     # target=$(get_target)
+    
+    echo patching "$(pwd)/ext/phar/build_precommand.php"
+    sed -i .bak '3i\
+ ini_set("pcre.jit", "0");
+' ext/phar/build_precommand.php
+
 
     # Build PHP
     echo "Running buildconfig..."

